profile/active.go

```diff
@@ -58,6 +58,13 @@ func addActiveProfile(profile *Profile) {
 	activeProfilesLock.Lock()
 	defer activeProfilesLock.Unlock()
 
+	// Mark any previous profile as outdated.
+	previous, ok := activeProfiles[profile.ScopedID()]
+	if ok {
+		previous.outdated.Set()
+	}
```

Comment:
`previous` and `ok` are not required in the outer scope so consider:
```suggestion
if previous, ok := activeProfiles[profile.ScopedID()]; ok {
    previous.outdated.Set()
}
```